app/src/org/commcare/fragments/CommCarePreferenceFragment.java

```diff
@@ -0,0 +1,131 @@
+package org.commcare.fragments;
+
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.support.annotation.CallSuper;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v7.preference.PreferenceFragmentCompat;
+import android.support.v7.preference.PreferenceManager;
+import android.support.v7.preference.PreferenceScreen;
+import android.util.Log;
+
+import org.commcare.CommCareApplication;
+import org.commcare.google.services.analytics.GoogleAnalyticsFields;
+import org.commcare.google.services.analytics.GoogleAnalyticsUtils;
+import org.javarosa.core.services.locale.Localization;
+import org.javarosa.core.util.NoLocalizedTextException;
+
+import java.util.Map;
+
+/**
+ * Common PreferenceFragment class from which all other preferences extend.
+ */
+public abstract class CommCarePreferenceFragment extends PreferenceFragmentCompat
+        implements SharedPreferences.OnSharedPreferenceChangeListener {
+
+    private static final String TAG = CommCarePreferenceFragment.class.getSimpleName();
+
+    @Override
+    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
+        GoogleAnalyticsUtils.reportPrefActivityEntry(GoogleAnalyticsFields.CATEGORY_CC_PREFS);
```

Comment:
If you look at all of the calls to `reportPrefActivityEntry()` that were removed in this PR, you'll notice that each preference activity used to invoke this call with its own specific category. Can you switch it back to doing that? Probably makes sense to do it in the same way that you've implemented setting a different title for each fragment